packages/util/src/constants.ts

```diff
@@ -261,3 +266,3274 @@ export const WETHAddress: Record<Network, Lowercase<TokenAddress>> = {
   [Network.mainnet]: toLower(tokens.mainnet.WETH),
   [Network.arbitrum]: toLower(tokens.arbitrum.WETH),
 };
+
+export const NEW_USER_ADDRESSES = [
```

Comment:
I think we should move this into a file that we fetch via HTTP instead of committing to the repo

Reply:
@matthew-garrett can you fetch the list from here:
https://registry.notional.finance/newcomer-boost.json